projects/plugins/jetpack/changelog/add-get-current-product

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: other
+
+Add get_current_product() and current_product_is_business_or_higher() to the WPcom_Admin_Menu.
```

Comment:
Was `current_product_is_business_or_higher()` deleted at some point?

Reply:
Yes. I'll remove that. We moved the actual plan check into the filter here D69877-code.